Source/WorkersES6/decodeDraco.js

```diff
@@ -272,16 +272,36 @@ function decodePointCloud(parameters) {
   return result;
 }
 
+function findQuantizationTransform(array, numComponents) {
+  var min = [];
+  var max = [];
+  var i, j;
+  for (i = 0; i < numComponents; ++i) {
+    min.push(array[i]);
+    max.push(array[i]);
+  }
+  for (i = 0; i < array.length; i += numComponents) {
+    for (j = 0; j < numComponents; ++j) {
+      min[j] = Math.min(array[i + j], min[j]);
+      max[j] = Math.max(array[i + j], max[j]);
+    }
+  }
+  var range = max[0] - min[0];
+  for (i = 1; i < numComponents; ++i) {
+    range = Math.max(max[i] - min[i], range);
```

Comment:
This is not what I expected - the [draco source code](https://github.com/google/draco/blob/85a614fa6639d155f47ec68a776258a0e28b918f/src/draco/attributes/attribute_quantization_transform.h#L96-L97) say `range` is the "max delta over all components." I first thought this would mean the magnitude of `max - min`, but doing it this way matches the range returned by the decoder in main.

Reply:
Oh, I didn't see this, maybe ignore my comment in the PR review :sweat_smile: